### PR TITLE
[new release] color-brewery (0.1)

### PR DESCRIPTION
**Pull request by Chris00**
Offer colors palettes and functions to brew colors

- Project page: <a href="https://github.com/Chris00/color_brewery">https://github.com/Chris00/color_brewery</a>
- Documentation: <a href="https://Chris00.github.io/color_brewery/doc">https://Chris00.github.io/color_brewery/doc</a>

##### CHANGES:

- Initial release.

### CODE REVIEW

**Comment by Chris00** [comment]
@avsm @kit-ty-kate It may be interesting that the CI also checks a flambda compiler — for example, for this project, a special workaround is needed for the stack size.

**Comment by kit-ty-kate** [comment]
Sounds fair. I'll make sure to add this to the new CI due to replace Travis and datakit-ci in a few months.

File: packages/color-brewery/color-brewery.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+build: [
+  ["dune" "subst"] {pinned}
+  ["sh" "-c" "ulimit -s 16384 && dune build -p %{name}% -j %{jobs}%"
+   ] {os = "linux"}
+  ["dune" "build" "-p" name "-j" jobs] {os != "linux"}
+]
+depends: [
+  "ocaml" {>= "4.03"}
+  "gg"
+  "dune"
```
Comment (kit-ty-kate) [inline packages/color-brewery/color-brewery.0.1/opam]:
```suggestion
  "dune" {>= "1.3"}
```
According to the `dune-project` file, this package requires at least dune 1.3

Comment (Chris00) [reply packages/color-brewery/color-brewery.0.1/opam]: It does not really (just was generated by that version of Dune).  Will upload a new tarball saying `1.0`.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**